RUFAS/task_manager.py

```diff
@@ -436,6 +436,37 @@ def handle_single_simulation_run(args: Dict[str, Any], output_manager: OutputMan
         simulator.simulate()
         output_manager.add_log("Simulation completed", "Simulation completed", info_map)
 
+    @staticmethod
+    def _handle_end_to_end_testing(
+        args: Dict[str, Any],
+        input_manager: InputManager,
+        output_manager: OutputManager,
+        task_id: str,
+        produce_graphics: bool,
+    ) -> None:
+        """Runs end-to-end testing routine."""
+        info_map = {
+            "class": TaskManager.__name__,
+            "function": TaskManager._handle_end_to_end_testing.__name__,
+            "task_id": task_id,
+            "produce_graphics": produce_graphics,
+        }
+
+        output_manager.add_log("End-to-end testing", "Starting end-to-end testing simulation.", info_map)
```

Comment:
```suggestion
        output_manager.add_log("End-to-end testing", "Starting simulation for end-to-end testing.", info_map)
```
It implies that we are simulating e2e